tests/test_ssds.py

```diff
@@ -255,6 +255,16 @@ def test_tags(self):
             blobstore.put_tags(bucket_name, key, tags)
             self.assertEqual(tags, blobstore.get_tags(bucket_name, key))
 
+    def test_exists(self):
+        key = f"{uuid4()}"
+        tests = [("aws", S3_SSDS.blobstore, S3_SSDS.bucket, self._put_s3_obj),
+                 ("gcp", GS_SSDS.blobstore, GS_SSDS.bucket, self._put_gs_obj)]
+        for replica_name, blobstore, bucket_name, upload in tests:
+            with self.subTest(replica=replica_name):
+                self.assertFalse(blobstore.exists(bucket_name, key))
+                upload(bucket_name, key, b"")
```

Comment:
Should this clean up after upload?

Reply:
Lifecycle policies on the test buckets delete everything after a day.